CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 StorIO Change Log
 ==========
 
+## Version 1.12.0
+
+_2016_12_7_
+
+* Support for **AutoValue** and **Kotlin data classes** in StorIOSQLiteProcessor and StorIOContentResolverProcessor. Many thanks to [@geralt-encore](https://github.com/geralt-encore)!
```

Comment:
No, it didn't. The problem was that in Kotlin you don't work with directly with fields - you work with properties. And you couldn't apply column annotation on backed field property because it's private, which was limitation of the previous annotation processor implementation.